keyboards/xd87/keymaps/rgb_default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Louwii
```

Comment:
This keymap should be named `default_rgb` or `default_underglow`, actually. I'm leaning towards the latter as there are two RGB-related features (Underglow and Matrix) and that would help to disambiguate.

Reply:
Renamed it `default_underglow`